test/powershell/Language/Scripting/SuppressAnsiEscapeSequence.Tests.ps1

```diff
@@ -3,12 +3,24 @@
 
 Describe '$env:__SuppressAnsiEscapeSequences tests' -Tag CI {
     BeforeAll {
+        $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+
+        if ($IsWindows) {
+            $osInfo = [System.Environment]::OSVersion.Version
```

Comment:
Can you rely on `$host.ui.SupportsVirtualTerminal` instead?

Reply:
I think so. that will be simpler.